models/mocks/Engine.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.43.2. DO NOT EDIT.
+// Code generated by mockery v2.21.4. DO NOT EDIT.
```

Comment:
The Make recipe for `generate`, uses mockery [v2.43.2](https://github.com/onflow/flow-evm-gateway/blob/main/Makefile#L27). I am not sure how you got `v2.21.4`, but I would suggesting running `make generate` locally, and making sure that:
```bash
$ mockery --version

03 Jul 24 13:52 EEST INF couldn't read any config file version=v2.43.2
03 Jul 24 13:52 EEST INF Starting mockery dry-run=false version=v2.43.2
03 Jul 24 13:52 EEST INF Using config:  dry-run=false version=v2.43.2
v2.43.2
```

Reply:
I did run make generate but the script has a mistake in download command, it should be install, otherwise it just uses an existing instalation.

Reply:
Oh, nice catch :100: